co-simulation/fed/mosaic-carla/pom.xml

```diff
@@ -79,6 +85,18 @@
                                 <skipTests>true</skipTests>
                             </configuration>
                         </plugin>
+                        <plugin>
+                          <artifactId>maven-assembly-plugin</artifactId>
```

Comment:
What is this section for?

Reply:
Wait is this packaging all dependent jar's into a single Jar? Does that mean the resource bundle stuff is no longer necessary.

Reply:
This was a test that should be removed, sorry for the confusion